docs/readmes/proposals/p020_n7_Interface_n_network_initiated_session_modification.md

```diff
@@ -0,0 +1,165 @@
+---
+id: p020_n7_Interface_n_network_initiated_session_modification
+title: SMF to PCF Interface and Network Initiated Session Modification
+hide_title: true
+---
+# Overview
+
+*Author: @pmishra-wavelabs*\
+*Last Updated: 07/12/2021*
+
+## Background and Objective 
+This document provides an approach for the following 
+- Implementing N7 interface towards PCF through Federation Gateway
+  -- Implement 5G Services Based Interface (SBI) for Federation Gateway
+  -- Feg Relay for SM Policy control create, update, delete and update-notify
+  -- Feg Session Proxy for SM Policy control create, update, delete and 
+     update-notify 
+- Implementing Network initiated session modification
+  -- Network Initiated Session Policy association update
+  -- Network initiated session Policy association termination
```

Comment:
Note FEG is just a translator from GRPC into 3gpp world. session_proxy this translator for the Gx/Gy interface for LTE so it is directly tight to diameter. I think N7 will require a brand new service

Does pcrf and PCF share any kind of messaging protocol/structure?

Are the messages sent from sessiond to FEG exactly the same for LTE and 5g?